backend/tests/test_utils/proposal_utils.go

```diff
@@ -138,16 +116,37 @@ func (otu *OverflowTestUtils) GenerateDraftProposalStruct(
 	proposal.Community_id = communityId
 	proposal.Start_time = time.Now().AddDate(0, 1, 0)
 	proposal.End_time = time.Now().Add(30 * 24 * time.Hour)
-
-	proposal.Status = nil
 	return &proposal
 }
 
 func (otu *OverflowTestUtils) GenerateCancelProposalStruct(
 	signer string,
 	proposalId int,
```

Comment:
looks like proposalId isn't being used in this function or the two below. we should either include it in the `Proposal` model, or omit it from the args